packages/firebase-auth/src/index.ts

```diff
@@ -142,10 +143,12 @@ export const verifySessionCookieFirebaseAuth = (
       })
       setFirebaseToken(c, idToken)
     } catch (err) {
-      const res = c.redirect(config.redirects.signIn)
-      throw new HTTPException(res.status, {
+      const status = 302
+      const res = c.redirect(config.redirects.signIn, status)
+      throw new HTTPException(status, {
         res,
         message: `failed to verify the requested firebase token: ${String(err)}`,
```

Comment:
Is `${String(err)}` necessary? I think it may not be needed if passing an `err` to a `cause`.

Reply:
@yusukebe Thanks!
This is for backward compatibles. Until now, there was no way to determine a root error, only whether it was contained in a string. Removing this process breaks existing application logic.

So I left it as it was!

Reply:
Thank you for the explanation! Understood well! Let's go with it.